.github/workflows/smoketest.yml

```diff
@@ -449,3 +450,86 @@ jobs:
         sudo update-locale
         tar --zstd -xf install.tgz
         install/opt/HPCCSystems/bin/unittests
+
+  # NB: this doesn't really need the post-setup data files included in install.tgz
+  # but as this is relatively quick and in parallel with others, it probably doesn't matter
+  eclwatch-ui-tests:
+    needs: build-and-setup
+    if: needs.build-and-setup.outputs.cache-hit != 'true' && github.event_name == 'pull_request'
+    timeout-minutes: 10
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/download-artifact@v2
+      with:
+        name: installsetup-artifact
+
+    - name: Check-ECLWatch-UI-Test-Directory
+      run: |
+        tar --zstd -xf install.tgz
+        if [[ ! -d install/test-ui ]]
```

Comment:
trivial: woud have been enough just to test [[ ! -d install/test-ui/tests ]] , simplifying/reducing script.

Reply:
Ok